frontend/public/components/graphs/prometheus-poll-hook.ts

```diff
@@ -1,16 +1,12 @@
-/* eslint-disable react-hooks/exhaustive-deps */
-import { useCallback, useState } from 'react';
-
-import { usePoll, useSafeFetch } from '../utils';
+import { useURLPoll, URL_POLL_DEFAULT_DELAY } from '../utils';
 import { getPrometheusURL, PrometheusEndpoint } from './helpers';
 import { PrometheusResponse } from '.';
 
-const DEFAULT_DELAY = 15000; // 15 seconds
 const DEFAULT_SAMPLES = 60;
 const DEFAULT_TIMESPAN = 60 * 60 * 1000; // 1 hour
 
 export const usePrometheusPoll = ({
-  delay = DEFAULT_DELAY,
+  delay = URL_POLL_DEFAULT_DELAY,
```

Comment:
no need to define the default value as url-poll hook does that.
```suggestion
  delay,
```

Reply:
+1